src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -961,6 +961,65 @@ public override SyntaxTree SyntaxTree
 
         internal override IOperation GetOperationWorker(CSharpSyntaxNode node, GetOperationOptions options, CancellationToken cancellationToken)
         {
+            var bindingRoot = GetBindingRootOrInitializer(node);
```

Comment:
>var [](start = 12, length = 3)

Please spell out the type here and below. In compilers, type can be omitted only if it explicitly mentioned in the initializer. For example, ```var builder = ArrayBuilder<int>.GetInstance();```

Reply:
sure